README.md

```diff
@@ -19,8 +19,6 @@ All `nagios_*` types default to having a `target` under `/etc/nagios/`, but shou
 <a id="development"></a>
 ## Development
 
-Puppet Labs modules on the Puppet Forge are open projects, and community contributions are essential for keeping them great. We can't access the huge number of platforms and myriad of hardware, software, and deployment configurations that Puppet is intended to serve.
+When the `nagios_*` types were removed from Puppet in Puppet Platform 6 and extracted to this module, they were effectively deprecated and are now no longer under active develepment.
 
-We want to keep it as easy as possible to contribute changes so that our modules work in your environment. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
-
-For more information, see our [module contribution guide.](https://docs.puppetlabs.com/forge/contributing.html)
+While it's possible to follow our usual contribution guidelines for modules, we do not currently have a team dedicated to reviewing and merging pull requests for this repository and your PR could sit indefinitely waiting for review. As an alternative you can fork this repository, and make changes to your own forked version.
```

Comment:
How about 'While this repo is open for PRs following our usual contribution guidelines for modules,'?  It sounds a bit clearer to me what is meant initially, but maybe it also sounds more open than what we want.

Reply:
Yeah, I also dug around the settings and I couldn't quite figure it out but I almost wonder if there's a way we can just turn off PRs for this without archiving? or PRs that don't come from the puppetlabs org? @shrug do you know how to do something like that with your fancy admin access?